src/test/unit/controllers/SubscriptionCaseSearchController.test.ts

```diff
@@ -0,0 +1,129 @@
+import sinon from 'sinon';
+import { Response } from 'express';
+import SubscriptionCaseSearchController from '../../../main/controllers/SubscriptionCaseSearchController';
+import fs from 'fs';
+import path from 'path';
+import {mockRequest} from '../mocks/mockRequest';
+import {SubscriptionCaseSearchRequests} from '../../../main/resources/requests/subscriptionCaseSearchRequests';
+
+const subscriptionCaseSearchController = new SubscriptionCaseSearchController();
+const rawData = fs.readFileSync(path.resolve(__dirname, '../mocks/subscriptionCaseList.json'), 'utf-8');
+const subscriptionCaseResult = JSON.parse(rawData);
+const stub = sinon.stub(SubscriptionCaseSearchRequests.prototype, 'getSubscriptionCaseDetails');
+
+const validCaseNo = 'ABC12345';
+
+describe('Subscription Case Search Controller', () => {
+  let i18n = {};
+  it('should render the search page', () => {
+
+    i18n = {
+      'subscription-urn-search': {},
+    };
+
+    const response = {
+      render: function () {
+        return '';
+      },
+    } as unknown as Response;
+    const request = mockRequest(i18n);
+
+    const responseMock = sinon.mock(response);
+
+    const expectedData = {
+      ...i18n['subscription-case-search'],
+    };
+
+    responseMock.expects('render').once().withArgs('subscription-case-search', expectedData);
+
+    subscriptionCaseSearchController.get(request, response);
+
+    responseMock.verify();
+  });
+
+  it('should render case search page if there are no matching results', () => {
+
+    stub.withArgs(validCaseNo).returns(null);
+
+    const response = { render: function() {return '';}} as unknown as Response;
```

Comment:
response is already set on the line 24 it doesn't need to be re-initialized within "it" scope if its unchanged